services/blockchain-indexer/shared/dataService/dpos/delegates.js

```diff
@@ -85,24 +96,26 @@ const computeDelegateStatus = async () => {
 		return isPunished;
 	};
 
+	logger.debug('Determine delegate status.');
```

Comment:
Do we need this log?

Reply:
Yes, I think its better to keep this debug log.